xml/installation-installation-designate-install_designate_BIND.xml

```diff
@@ -10,19 +10,18 @@
  <section>
   <title>Install DNS Service with BIND</title>
   <para>
-   &kw-hos; DNS Service and BIND can be installed together instead of the
-   default <emphasis role="bold">PowerDNS</emphasis> backend. BIND will be
-   deployed to one or more control planes clusters. The following configuration
-   example denotes where the BIND service is installed.
+   &kw-hos; DNS Service defaults to the BIND Backend if another backend is not
+	 configured for domain name service. BIND will be deployed to one or
+   more control planes clusters. The following configuration example denotes
+	 where the BIND service is installed.
```

Comment:
Please avoid adding tab characters. (Try running `sed -i 's/\t/ /g' FILE` for a quick fix here.)
There are similar tabbing issues in all the other files touched by this branch.

Reply:
strange. I didn't add any tabs. And they look like regular spaces in Atom, which I didn't add either. Do you see them in a text editor or just in the commits here on Github?
All files touched by this branch have had the sed command applied. In both Atom and Kate, they look off. (designate_BIND.xml; line 13 starts with 3 spaces, line 14 2 spaces, line 15 3, line 16 2 spaces).